Surveys/capture-device-information/capture-device-information.js

```diff
@@ -0,0 +1,31 @@
+(function() {
+    // Set a variable for each question this script will answer.
+    const userAgentField = document.querySelector('.user-agent-capture input');
+    
+    // The field could have been set by a default value from a Profile Property.
+    let prevUserAgent = userAgentField.value;
+
+    function captureUserAgent() {
+        let currUserAgent = navigator.userAgent;
+
+        if (currUserAgent) {
+
+            if (prevUserAgent !== currUserAgent) {
+
+                // Set the value and hide the field from the user.
+                userAgentField.value = currUserAgent;
+                userAgentField.readOnly = true;
+                userAgentField.hidden = true;
+                
+                // Checkbox Frontend uses AngularJS, we need to force a DOM re-render.
```

Comment:
We are using Angular, not AngularJS. It is a different framework.

Reply:
@brendanorourke Should be good to merge after you address this minor comment!

Reply:
@brendanorourke I can make the edit if you'd like, just noticed this has been sitting open for a long time 😄